sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/AADAuthenticationFilterAutoConfiguration.java

```diff
@@ -71,8 +69,6 @@ public AADAuthenticationFilterAutoConfiguration(AADAuthenticationProperties prop
     @Bean
     @ConditionalOnMissingBean(AADAuthenticationFilter.class)
     @ConditionalOnExpression("${spring.cloud.azure.active-directory.session-stateless:false} == false")
-    // client-id and client-secret used to: get graphApiToken -> groups
-    @ConditionalOnProperty(prefix = PROPERTY_PREFIX, value = { "client-id", "client-secret" })
```

Comment:
why delete this?
will "client-secret" be deleted?

Reply:
client-id and client-secret can be got from global property.